components/AriaTable/AriaTable.tsx

```diff
@@ -67,8 +68,28 @@ export const Td = forwardRef<
   ElementRef<typeof StyledTd>,
   Omit<ComponentProps<typeof StyledTd>, 'css'> &
     VariantProps<typeof StyledTd> &
-    VariantProps<typeof TableTd> & { css?: CSS }
->((props, ref) => <StyledTd ref={ref} role="cell" {...props} />);
+    VariantProps<typeof TableTd> & { css?: CSS; colSpan?: number }
+>(({ colSpan, css, ...props }, ref) => {
+  const colSpanCss = useMemo(
+    () =>
+      colSpan
+        ? {
+            width: `${Math.max(colSpan || 0, 0) * 100}%`,
+            float: 'left',
```

Comment:
![interesting-megan](https://user-images.githubusercontent.com/38889364/158506679-37f722d2-f08b-45d6-ae4f-31fbd90457a1.gif)

Reply:
I don't feel comfortable using `float` but at the same time I didn't find a way to achieve this with `display: table-cell`.

Reply:
Yup, this is a tweak...

No documentation on how to achieve the behaviour another way. `column-span: 'all'` doesn't work, it was my best  bet until I found this.

I did a very thorough research before resorting to this solution

Reply:
Yeap, I tried this `column-span` too, no luck. Let's keep it like this until we find a better approach and handle it in another PR.